### DISCUSSION TITLE
Looking for npx create-next-app template that matches "Docs" on nextjs.org/docs

### DISCUSSION THREAD

**Discussion by shoutmarble** [vercel/next.js#70154] [Help] [answered]
### Summary

I'm looking for the "[Docs](https://github.com/vercel/next.js/tree/canary/examples/)" template that matches [nextjs.org/docs](https://nextjs.org/docs) in the Git Directory:
[examples](https://github.com/vercel/next.js/tree/canary/examples/)

So, I can begin a Docs starter next.js app with `npx create-next-app`
`npx create-next-app --example cms-builder-io cms-builder-io-app`  

Which template is being used for `nextjs.org/docs?`

I found these 2 templates at:

[SanDocs - Free Next.js Product Documentation Template](https://nextjstemplates.com/templates/sandocs-nextjs-docs-template)

[Nextra - Next.js Docs Site Template](https://nextjstemplates.com/templates/nextra-docs)

Either, of these two will work perfectly for my purposes.

I just have an itch to use the exact template template found on "[Docs](https://nextjs.org/docs)" 

Thank you

**Comment by Juneezee** [comment, marked as answer]
> The source code for rendering the docs isn't open source, just the docs content in this repo.

See https://github.com/vercel/next.js/discussions/54936#discussioncomment-6899058
